src/elements/Embed/IframelyEmbed.scss

```diff
@@ -9,4 +9,8 @@
         border: 1px solid $color-error;
         border-radius: $border-radius-base;
     }
+
+    blockquote {
+        border-left: none;
+    }
```

Comment:
There was an issue with displaying TikTok embeds, which had a blockquote wrapper. The block had unwanted left border inherited from global styles.